### DISCUSSION TITLE
I keep getting this error, how do i fix it.

### DISCUSSION THREAD

**Discussion by Cole-1010101010** [berrystephenw/OctoText#51] [Q&A]
![image](https://user-images.githubusercontent.com/82060805/113775677-f0fd4100-96dd-11eb-8690-0f3133f7f7eb.png)

**Comment by berrystephenw** [comment]
Hi Cole, you are getting this message because the email account you are using is either set up incorrectly (wrong user or password) or the email provider is requiring an app password. Gmail is famous for this. The FAQ has instructions for how to get around this (top of the discussion page). If that doesn't help, your octoprint log file would help me figure it out. Just attach it here or email it to me (sberry@gravity-performance.com)

**Reply by Cole-1010101010** [reply]
is the office 365 not working because i use outlook live, not outlook 365?  also how do i get my octoprint log file?

**Reply by Cole-1010101010** [reply]
P.S. My end goal is to have texts sent to me, not emails

**Reply by berrystephenw** [reply]
I use outlook live also - it's just what they call the free email service. The settings for OctoText will need to change to match the email account that you setup. The defaults for gateway and port should work for you - but the email address needs to change to whatever you picked. 
![image](https://user-images.githubusercontent.com/44931130/113892453-fe750280-9793-11eb-8698-7513ed41b318.png)

At some point I had to setup an "app" password because Microsoft thought I might be spamming people, but you aren't there yet since you haven't gotten to the point of logging into your email account.

The way OctoText works is that it uses your email account to send an email to your phone providers SMS/MMS gateway. Your phone provider will send the "email" to your phone - which shows up as a text. In the picture above you need to enter your phone number where I've blacked it out, and find the texting gateway for your provider. The readme on the front page should give you a gateway to try if you are in the US. 

I hope that helps. Oh, the log files are found in the settings menu (the wrench icon top of the screen) here:
![image](https://user-images.githubusercontent.com/44931130/113894112-b060fe80-9795-11eb-968e-b605140eb2cc.png)

click the highlighted download button.

**Reply by Cole-1010101010** [reply]
i think i fixed it, when my print started it sent me a tect message. I had to put my number as 1234567890@vtext.com instead of just the number. Also, when i press test message, it doesnt send but it did send when my print started so. 🤷

**Comment by berrystephenw** [comment]
Weird, that shouldn’t have worked at all! Unless you left the destination blank?

Stephen Berry
Hardware Engineering Manager
GSSI
40 Simon St
Nashua, NH 03060
________________________________
